app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -545,9 +568,9 @@ class TabTrayView(
             // We add the offset, because the layoutManager is initialized with `reverseLayout`.
             // We also add 1 to display the tab item above the selected browser tab.
             val recyclerViewIndex = selectedBrowserTabIndex +
-                collectionsButtonAdapter.itemCount +
-                syncedTabsController.adapter.itemCount +
-                1
+                    collectionsButtonAdapter.itemCount +
```

Comment:
Is this change needed? Seems like only the formatting was changed.
I'd prefer sticking as much as possible to the original code to preserve commit history and so to have an easier life later when wearing the code archaeologist hat.